### PR TITLE
feat(terminal): add first version

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-1034

Add a first version without pods dropdown, with some other improvement:
+ Improve Skeleton height for ServiceActionToolbar, EnvironmentActionToolbar and ClusterActionToolbar to avoid loading glitch
+ Update Toast to authorise string for component status
+ Update `useReactQueryWsSubscription` hook to allow binary response

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Take rest sometimes

File: libs/domains/services/feature/src/lib/service-terminal/service-terminal.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+export function ServiceTerminal({
+  organizationId,
+  clusterId,
+  projectId,
+  environmentId,
+  serviceId,
+}: ServiceTerminalProps) {
+  const { setOpen } = useContext(ServiceTerminalContext)
+  const [attachAddon, setAttachAddon] = useState<AttachAddon | undefined>(undefined)
+  const [haveMessage, setHaveMessage] = useState(false)
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-terminal/service-terminal.tsx]: https://github.com/Qovery/console/pull/1245/commits/b43ac32951b27bff9f4de6ecc56fb73a35ab500f

File: libs/domains/services/feature/src/lib/service-action-toolbar/service-action-toolbar.tsx [hunk 2]
```diff
@@ -579,15 +583,41 @@ export function ServiceActionToolbar({ environment, serviceId }: { environment:
...
-          }}
+          }
         >
           <Icon iconName="scroll" />
         </ActionToolbar.Button>
       </Tooltip>
+      {service.serviceType !== 'DATABASE' && (
+        <Tooltip content="Qovery cloud shell">
+          <ActionToolbar.Button
+            onClick={() => {
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-action-toolbar/service-action-toolbar.tsx:598]: I need to display the button if the service is running only, I think it's better if I have the condition directly in this toolbar no? a bit like we have for actions on this component

[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/domains/services/feature/src/lib/service-action-toolbar/service-action-toolbar.tsx]: https://github.com/Qovery/console/pull/1245/commits/40bcc280b0ddb845c97240303cb73a5159a584ef

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**